packages/platform/src/batch.rs

```diff
@@ -16,7 +16,7 @@ pub type ReplyId = u64;
 #[derive(Default)]
 #[cfg_attr(
     any(debug_assertions, test, feature = "testing"),
-    derive(Debug, PartialEq)
+    derive(Debug, PartialEq, Eq)
 )]
```

Comment:
I'd suggest dropping the conditional compilation for those. Either way they will be removed either by the compiler or linker.
```suggestion
#[derive(Debug, PartialEq, Eq, Default)]
```